users/src/main/java/io/crate/operation/user/UserManagerService.java

```diff
@@ -63,6 +63,13 @@
         }
     };
 
+    private static final Consumer<User> ENSURE_PRIVILEGE_USER_NOT_SUPERUSER = user -> {
```

Comment:
no because this is related to privileges, not drop like the above (message differs